src/motor_driver/canmotorlib.py

```diff
@@ -36,6 +36,36 @@
                 "AXIS_DIRECTION" : -1
                 }
 
+# Working parameters for AK80-6 V1.1 firmware
+AK80_6_V1p1_PARAMS = {
+                "P_MIN" : -12.5,
+                "P_MAX" : 12.5,
+                "V_MIN" : -22.5,
+                "V_MAX" : 22.5,
+                "KP_MIN" : 0.0,
+                "KP_MAX" : 500,
+                "KD_MIN" : 0.0,
+                "KD_MAX" : 5.0,
+                "T_MIN" : -12.0,
+                "T_MAX" : 12.0,
+                "AXIS_DIRECTION" : -1
```

Comment:
@babel-robotics @vyas-shubham are you sure the axis direction is -1? I thought this was addressed already.

Reply:
No, you are right Shivesh. The axis direction should indeed be 1.